chapters/operatorsandexpressions.tex

```diff
@@ -821,28 +818,14 @@ \subsubsection{spatialDistribution}\label{spatialdistribution}
                                    initialPoints = {0.0, 1.0},
                                    initialValues = {0.0, 0.0});
 \end{lstlisting}
-where \lstinline!in0!, \lstinline!in1!, \lstinline!out0!, \lstinline!out1!, \lstinline!x!, \lstinline!v! are all subtypes of Real,
-\lstinline!positiveVelocity! is a Boolean, \lstinline!initialPoints! and \lstinline!initialValues! are
-arrays of subtypes of Real of equal size, containing the y coordinates
-and the $z$ values of a finite set of points describing the initial
-distribution of $z(y, \mathit{t0})$. The \lstinline!out0! and \lstinline!out1! are given by
-the solutions at $z(0.0, t)$ and $z(1.0, t)$; and \lstinline!in0! and \lstinline!in1!
-are the boundary conditions at $z(0.0, t)$ and $z(1.0, t)$ (at
-each point in time only one of in0 and in1 is used). Elements in the
-initialPoints array must be sorted in non-descending order. The operator
-can not be vectorized according to the vectorization rules described in
-\cref{scalar-functions-applied-to-array-arguments}. The operator can be vectorized only with respect to the
-arguments in0 and in1 (which must have the same size), returning
-vectorized outputs out0 and out1 of the same size; the arguments
-initialPoints and initialValues are vectorized accordingly.
+where \lstinline!in0!, \lstinline!in1!, \lstinline!out0!, \lstinline!out1!, \lstinline!x!, \lstinline!v! are all subtypes of \lstinline!Real!, \lstinline!positiveVelocity! is a \lstinline!Boolean!, \lstinline!initialPoints! and \lstinline!initialValues! are arrays of subtypes of \lstinline!Real! of equal size, containing the y coordinates and the $z$ values of a finite set of points describing the initial distribution of $z(y, \mathit{t0})$.  The \lstinline!out0! and \lstinline!out1! are given by the solutions at $z(0.0, t)$ and $z(1.0, t)$; and \lstinline!in0! and \lstinline!in1! are the boundary conditions at $z(0.0, t)$ and $z(1.0, t)$ (at each point in time only one of in0 and in1 is used).  Elements in the \lstinline!initialPoints! array must be sorted in non-descending order.  The operator can not be vectorized according to the vectorization rules described in \cref{scalar-functions-applied-to-array-arguments}.  The operator can be vectorized only with respect to the arguments in0 and in1 (which must have the same size), returning vectorized outputs out0 and out1 of the same size; the arguments \lstinline!initialPoints! and \lstinline!initialValues! are vectorized accordingly.
```

Comment:
Two unwrapped cases of in0 and in1 still remain.

Reply:
Fixed (also `out0` and `out1`).  Resolving.